src/corelibs/U2Formats/src/BAMUtils.cpp

```diff
@@ -346,7 +346,7 @@ static QString createNumericSuffix(int n) {
 
 static void bamSortBlocks(int n, int k, bam1_p* buf, const QString& prefix, const bam_header_t* h) {
     QString sortedFileName = n < 0 ? prefix + ".bam" : prefix + "." + createNumericSuffix(n) + ".bam";
-    const char* mode = n < 0 ? "w" : "w1";
+    const char* mode = n < 0 ? "w" : "w+";
```

Comment:
`w1` comes from the original samtools package.

I would use keep a simple `w` for both cases (n<0 & n>=0)
or
pass `w1` to bam_dopen and `w` to BAMUtils::openFile

There is no reason to use `w+` here

Reply:
OK